factory/templates/locale.conf.in

```diff
@@ -0,0 +1 @@
+LANG="{{ SYSTEMD_DEFAULT_LOCALE }}"
```

Comment:
Do we need a license header? Other config files we ship have them - also maybe a comment ` # File generated by ...`? Mine generated from Debian has a similar comment

Reply:
I am unsure whether other parsers support comments here. I checked Fedora and the [systemd docs](https://www.freedesktop.org/software/systemd/man/locale.conf.html#Example) and didn't see any comment (on a Debian system I don't have such file, what generated it?). `systemd-firsboot` as well does not seem to inject any comment.

If we feel it is safe to have comments in there, I can add the generator/owner. A license is not needed IMHO, there is no alternative arrangement/approach for these few bytes really.

Reply:
I think it's fine not to add the header here. This file is likely to be modified by adding more lines, by systemd-localed or even by some image modification scripts, and then the comment that this file was "created by systemd" will be silly.

Reply:
Oh woops, conflicting push from my side then. On the second commit, should I keep the docs changed or throw the whole commit out?

Reply:
Hmm, maybe say something like "This is a fallback locale.conf file provided by systemd." ?